api-docs/cloud/ref.yml

```diff
@@ -11039,35 +11038,135 @@ paths:
         - System information endpoints
   /api/v2/scripts:
     get:
+      description: >
+        Retrieves a list of
+        [scripts](/influxdb/cloud/api-guide/api-invokable-scripts/).
```

Comment:
```suggestion
        [invokable scripts](/influxdb/cloud/api-guide/api-invokable-scripts/).
```

Reply:
Let's keep `scripts` as they're likely to decouple from the invokable-ness.